client/src/components/RandomFactCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Typography, CardContent, Card, CardMedia, Button, CardActions } from '@mui/material';
+
+interface RandomFactProps {
+  fact: string;
+}
+
+const RandomFactCard: React.FC<RandomFactProps> = ({ fact }) => {
+  return (
+    <Card sx={{ display: 'flex', flexDirection: 'column', width: 300, height: 400 }}>
+      <CardMedia
+        component="img"
+        height="150"
+        /* NOTE: The url is appended with a unique query string to
+                 circumvent caching. Since the url is the same 
+                 otherwise, browsers would only generate 1 image. */
+        image={`https://picsum.photos/300/200?${Date.now()}`}
+        alt="A randomly generated image"
+      />
+      <CardContent sx={{ maxHeight: 200, overflowY: 'auto' }}>
+        <Typography variant="h6" component="h6" color="text.secondary" sx={{ fontSize: '14' }}>
```

Comment:
Suggestion: Maybe we can set the `fontSize` for the `body1` variant in MUI's theme, unless we explicitly want a different font size here? :)

Reply:
A-ha! Yes, we could. I can't remember quite exactly what the thought process that lead me to leaving this in was... At the time, I thought it was a nice subtle misdirection/"bug" (kinda similar to the `useHistory` issues or `CSSBaseline` usage in the previous version) but looking at it again, I don't like it myself either. 

I'll get rid of it 👍🏼

Reply:
Should be resolved with the new pushes, please have a look and confirm 👍🏼